server/views/pages/appointments/create-and-edit/schedule.njk

```diff
@@ -252,11 +252,13 @@
         {% for scheduledEvent in scheduledEvents %}
             {% set scheduledEventRows = (scheduledEventRows.push([
                 {
-                    text: timeText(scheduledEvent)
+                    text: timeText(scheduledEvent),
+                     classes: "no-break"
```

Comment:
I don't think the class is need here as well, is it?

Reply:
We need this classes otherwise its using 
.govuk-table__cell {
    word-break: break-word;
}

When we use the class .no-break it overrides